docs/scenarios_authoring/Intro.md

```diff
@@ -86,7 +87,7 @@ If the event which arrived to some node originally came from the Kafka topic, th
 Consult Kafka [documentation](https://kafka.apache.org/33/javadoc/org/apache/kafka/clients/consumer/ConsumerRecord.html) for the exact meaning of those fields. 
 
 
-The `#meta` variable carries meta information about the currently executed scenario. The following meta information elements are available:
+The `#meta` variable carries meta information about the scenario in execution. The following meta information elements are available:
```

Comment:
Question to nu-devs: does this tag access "dynamic" data, available only when scenario is being executed, or only static data such as scenario name? If it accesses only static data maybe here we should write "about the current scenario" without mentioning execution.

Reply:
What kind of improvement does this formulation bring ("scenario in execution")? Is it really more clear?

Reply:
I see that in meta we can access for instance #meta.properties.checkpointIntervalInSeconds, so it seems that this is an "dynamic" data.

We have here situation similar to "process" and "program" distinction. I would go with
"The `#meta` variable carries meta information about the execution of scenario"

Reply:
I have changed the wording to underline the fact that meta is a dynamically allocated variable.